### PR TITLE
add atmospherejs provider

### PR DESCRIPTION
**Pull request by mickythump**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ s -b w3m cats
 ## Supported Providers
 
 * amazon
+* atmosphere
```
Comment (zquestz) [inline README.md]: should be atmospherejs

File: providers/atmospherejs/atmospherejs.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("atmospherejs", &AtmosphereJS{})
+}
+
+// BaiduProvider adheres to the Provider interface.
+type AtmosphereJS struct {
```
Comment (zquestz) [inline providers/atmospherejs/atmospherejs.go]: Our naming convention is "AtmosphereJSProvider"

File: s.go [hunk 3]
```diff
@@ -27,7 +28,7 @@ import (
 	_ "github.com/zquestz/s/providers/ietf"
 	_ "github.com/zquestz/s/providers/kickasstorrents"
 	_ "github.com/zquestz/s/providers/libgen"
-	_ "github.com/zquestz/s/providers/nhaccuatui"    
+	_ "github.com/zquestz/s/providers/nhaccuatui"
```
Comment (zquestz) [inline s.go]: Looks like some cruft from another provider.

File: providers/atmospherejs/atmospherejs.go [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("atmospherejs", &AtmosphereJSProvider{})
+}
+
+// BaiduProvider adheres to the Provider interface.
```
Comment (zquestz) [inline providers/atmospherejs/atmospherejs.go]: Not Baidu =)

File: providers/atmospherejs/atmospherejs.go [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+type AtmosphereJSProvider struct {
+}
+
+// BuildURI generates a search URL for Google.
```
Comment (zquestz) [inline providers/atmospherejs/atmospherejs.go]: Not Google =)

File: s.go (hunk 3 above)
Comment (zquestz) [inline s.go]: NM you were fixing whitespace =)

Comment (mickythump) [inline s.go]: Atom did it for me probably :)

**Comment by zquestz** [comment]
This looks good to merge after rebase to single commit.

**Comment by mickythump** [comment]
I'm closing this one as well. I've messed up the whole squashing process, since it's been my first time doing it. I really should practice this on my repos first.

**Pull request closed without merging**